test/libsolidity/util/BytesUtils.cpp

```diff
@@ -206,6 +217,31 @@ string BytesUtils::formatString(bytes const& _bytes, size_t _cutOff)
 	return os.str();
 }
 
+std::string BytesUtils::formatFixedPoint(bytes const& _bytes, bool _signed, size_t _fractionalDigits)
+{
+	string value;
```

Comment:
Can you rename this to `decimal` to make it clear what it actually is - the decimal representation of the value?